disperser/dataapi/operator_handler.go

```diff
@@ -340,7 +357,7 @@ func (oh *OperatorHandler) CreateOperatorQuorumIntervals(
 }
 
 func (oh *OperatorHandler) getOperatorQuorumEvents(
-	ctx context.Context,
+	_ context.Context,
```

Comment:
curious on why we are are dropping ctx here?

Reply:
Lint. The `ctx` is not used.